### ISSUE TITLE
query staking validators ignores --limit

### ISSUE THREAD

**Issue by troian** [ovrclk/akash#981]

**Comment by boz** [comment]
I think this needs to be fixed upstream in [cosmos/cosmos-sdk](https://github.com/cosmos/cosmos-sdk)

**Comment by anilCSE** [comment]
@troian is it with CLI queries or REST?

**Comment by troian** [comment]
cli
```
❯ $AKASH query staking validators --limit 1
Error: unknown flag: --limit
Usage:
  akash query staking validators [flags]
```

**Comment by anilCSE** [comment]
This issue is fixed on upstream. Waiting for a new RC

**Comment by troian** [comment]
@anilCSE the fix is in 0.40.x, right?

**Comment by troian** [comment]
fix confirmed
